### ISSUE TITLE
settings issue

### ISSUE THREAD

**Issue by johnfa155** [Cobinja/CobiAnalogClock#13]
Hi, I have used Cobianalog clock on a previous Ubuntu, but much prefer mint but i have a problem with this I hope you can help, I like the desktop clock, but when I try to go in to properties to alter some settings I get the following message 

Execution of 'home/john/.local/share....../settings.py 10 failed:

failed to execute child proccess "home/john/......./analog-clock.de/3.0/settings.py" (permission denied)

I am using mint 18.3 Cinnamon 64 bit Linux kernel 4.10.0-42-generic if you can help please

Regards John

**Comment by Cobinja** [comment]
Is that file executable? If not, do "chmod +x settings.py" in a terminal

**Comment by johnfa155** [comment]
yes, every thing runs ok I just can't get to settings to alter anything, just get the message I have tried uninstalling and reinstalling but its the same

**Comment by johnfa155** [comment]
it says  chmod: cannot access  'settins.py' :no such file or directory
